lambda/src/main/resources/db/migration/V8__add-matches-table

```diff
@@ -0,0 +1,38 @@
+DROP TABLE "FFIDMetadata";
+CREATE TABLE "FFIDMetadata"
+(
+    "FFIDMetadataId" uuid NOT NULL,
+    "FileId" uuid NOT NULL,
+    "Software" text COLLATE pg_catalog."default" NOT NULL,
+    "SoftwareVersion" text COLLATE pg_catalog."default" NOT NULL,
+    "Datetime" timestamp with time zone not null,
+    "BinarySignatureFileVersion" text COLLATE pg_catalog."default" NOT NULL,
+    "ContainerSignatureFileVersion" text COLLATE pg_catalog."default" NOT NULL,
+    "Method" text COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT "FFIDMetadata_pkey" PRIMARY KEY ("FFIDMetadataId"),
+    CONSTRAINT "FFIDMetadata_Consignment_fkey" FOREIGN KEY ("FileId")
+        REFERENCES "File" ("FileId") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID
```

Comment:
Is this line necessary? We're creating a table from scratch, so it should be empty (and therefore valid) from the start.

Reply:
I've removed it and it still works fine, as you say because it's on an empty table. I've removed the other one as well.